src/cpp/r/R/Api.R

```diff
@@ -368,6 +376,15 @@
       url = NULL)
 })
 
+.rs.addApiFunction("writePreference", function(name, value) {
```

Comment:
Not sure we want to expose the preference functions as it would also require our documenting the names of all the UI preferences and create an alternate channel for mutating them that we don't control. What's the motivation for exposing these?